lib/import-export-visitor.js

```diff
@@ -717,6 +717,13 @@ function processRemoval(removal) {
   }
 }
 
+function safeKey(key) {
+  if (/^\d+$|^[_$a-zA-Z]\w*$/.test(key)) {
+    return key;
+  }
+  return JSON.stringify(key);
```

Comment:
This may be too safe. I don't think identifiers can be just numbers as they can be with keys.